resources/sdk/clisdkclient/templates/restclient.mustache

```diff
@@ -243,6 +269,48 @@ func authorize(c config.Configuration) (models.OAuthTokenData, error) {
 	return *oAuthTokenResponse, nil
 }
 
+func startCodeAuthServer(wg *sync.WaitGroup, uri string) {
+	address := strings.Replace(uri, "http://localhost", "", -1)
+	port := address[0:5]
+	path := strings.Replace(address, port, "", -1)
+	authCode := ""
+
+	getAuthCode := func(w http.ResponseWriter, r *http.Request) {
+		authCode = r.URL.Query().Get("code")
+		if authCode != "" {
+			codeAuthCode = authCode
+			wg.Done()
+			return
+		}
+	}
+
+	http.HandleFunc(path, getAuthCode)
+	log.Fatal(http.ListenAndServe(port, nil))
+}
+
+func openBrowserForLogin(wg *sync.WaitGroup, clientID string, redirectURI string) {
+	encodedURI := strings.Replace(redirectURI, ":", "%3A", -1)
```

Comment:
Pretty sure the "proper way" would be to generate the loginUrl with an unedited redirectURI and then use `url.Values.Encode()` to encode it. Example: https://golang.cafe/blog/how-to-url-encode-string-in-golang-example.html